README.md

```diff
@@ -1,111 +1,130 @@
-[c:geo](https://www.cgeo.org) is an open source, full-featured, always ready-to-go client for geocaching.com (unofficial) and offers basic support for other geocaching platforms.
+<img align="right" src="main/res/mipmap-xxhdpi/ic_launcher_round.png">
+
+# c:geo
+
+[c:geo](https://www.cgeo.org/) is an open-source, full-featured, always ready-to-go client for geocaching.com (unofficial).
+It also offers basic support for other geocaching platforms.
 It does not require a web browser or exports - just download and start right away.
 
-## You want to contribute?
+## Want to contribute?
 
-Perfect! Please **tell us in the [issue tracker](https://github.com/cgeo/cgeo/issues) before hacking** on your great new feature. It would be bad if you have implemented something great, but we can't include it because it doesn't fit the remaining architecture and code. You might also want to chat with the developers on channel #cgeo on the freenode IRC network.
+Perfect! Please **tell us in the [issue tracker](https://github.com/cgeo/cgeo/issues) before hacking** on your great new feature.
+It would be bad for you to have implemented something great but we can't include it because it doesn't fit the existing architecture and code.
+You might also want to chat with the developers on channel #cgeo on the freenode IRC network.
 
 ### Starting points for contribution
 
-You can also take a look onto the [project page](https://github.com/cgeo/cgeo/projects) of our repository.
-For example we have a collection of [urgent issues](https://github.com/cgeo/cgeo/projects/6) and a list of [beginner topics](https://github.com/cgeo/cgeo/projects/7) which collects issues, which might be suitable for your first contribution.
+You can also take a look at the [project page](https://github.com/cgeo/cgeo/projects) of our repository.
+For example, we have a collection of [urgent issues](https://github.com/cgeo/cgeo/projects/6) and a list of [beginner topics](https://github.com/cgeo/cgeo/projects/7).
+They collect issues that might be suitable for your first contribution.
 
 ## Project status
 
 [![Build Status](https://ci.cgeo.org/job/cgeo%20continuous%20integration/badge/icon)](https://ci.cgeo.org/job/cgeo%20continuous%20integration/)<br>
-[![Codacy Badge](https://api.codacy.com/project/badge/grade/3256314c8ba8457b9639bd2d4f4e7c91)](https://www.codacy.com/app/cgeo/cgeo)<br>
+[![Codacy Badge](https://api.codacy.com/project/badge/grade/3256314c8ba8457b9639bd2d4f4e7c91)](https://app.codacy.com/manual/cgeo/cgeo/)<br>
```

Comment:
The new link seems like it isn't working, while the old one still does. Can we stay with the old link?

Reply:
Sure